internal/pkg/daemon/bpfrecorder/bpfrecorder.go

```diff
@@ -545,68 +546,83 @@ func toStringByte(array []byte) string {
 
 func (b *BpfRecorder) processEvents(events chan []byte) {
 	b.logger.Info("Processing events")
+
+	// Allow up to 1000 goroutines to run in parallel
+	const workers = 1000
```

Comment:
Feels like something that could/should be configurable.

Reply:
Hm, how would users decide which value to choose here? I'm not sure about making it configurable for end users without confusing them.

Reply:
maybe this could be positively related with spod request and limits.

Reply:
@saschagrunert the worry was mostly around giving users a way to control this in case it got out of hand.
But I am happy to ahead as-is and review before next release.